src/hooks/tradingView/useTradingView.ts

```diff
@@ -119,7 +126,7 @@ export const useTradingView = ({
               orderLineToggleRef.current = tvWidgetRef.current.createButton();
               orderLineToggleRef.current.innerHTML = `<span>${stringGetter({
                 key: STRING_KEYS.ORDER_LINES,
-              })}</span> <div class="displayOrdersButton-toggle"></div>`;
+              })}</span> <div class="toggle"></div>`;
```

Comment:
nit: I wonder if it's worth factoring out ` toggleHtml()` helper

Reply:
i think as a followup im going to investigate just combining a lot of the logic here for the three settings that we have :P so combining the html here can be part of that!